pennylane/ops/qubit.py

```diff
@@ -2674,21 +2682,28 @@ class QubitSum(Operation):
 
     **Example**
 
-    The following circuit performs the ``QubitSum`` operation on a ``basis_state``.
-    Here we perform the modulo two sum :math:`1 \oplus 1 \oplus 0 = 0`:
+    The ``QubitSum`` operation maps the state :math:`|010\rangle` to :math:`|011\rangle`, with the
+    final wire holding the modulo-two sum of the first two wires:
 
     .. code-block::
 
+        input_bitstring = (0, 1, 0)
+
         @qml.qnode(dev)
         def circuit(basis_state):
             qml.BasisState(basis_state, wires = [0, 1, 2])
             qml.QubitSum(wires=[0, 1, 2])
-            return qml.probs(wires = 2)
+            return qml.probs(wires=[0, 1, 2])
+
+        probs = circuit(input_bitstring)
+        probs_indx = np.argwhere(probs == 1).flatten()[0]
+        bitstrings = list(itertools.product(range(2), repeat=3))
+        output_bitstring = bitstrings[probs_indx]
 
-        probs = circuit(np.array([1, 1, 0]))
-        bitstrings = tuple(itertools.product([0, 1], repeat=1))
-        indx = np.argwhere(probs == 1).flatten()[0]
-        output = bitstrings[indx]
+    The output bitstring is
+
+    >>> output_bitstring
+    (0, 1, 1)
```

Comment:
Would we want to add more here to reflect what was done in `QubitCarry`? I.e.

The action of ``QubitSum`` is to add wires ``1``, ``2``, and ``3``. The modulo-two result is output in wire ``3``. In this case, :math:`0 \oplus 1 \oplus 0 = 0` so we have:

`abc_sum = output_bitstring[2]`
`>>> abc_sum`

Reply:
Nice, done: e1eba7d0a4c084b85a0f51d164e548aa3133aa04